packages/conventional-commits-parser/lib/parser.js

```diff
@@ -86,7 +87,13 @@ function parser(raw, options, regex) {
     getCommentFilter(options.commentChar) :
     passTrough;
 
-  var lines = trimOffNewlines(raw).split(/\r?\n/).filter(commentFilter);
+  var rawLines = trimOffNewlines(raw).split(/\r?\n/);
+  var scissorIndex = rawLines.indexOf(SCISSOR);
```

Comment:
I'd be tempted to pull these few lines into a helper method; perhaps `truncateToScissor`?